contracts/test/TestCurveSwapper.sol

```diff
@@ -23,12 +23,12 @@ contract TestCurveSwapper is CurveSwapper {
         return _underlying;
     }
 
-    function test_swapIntoUnderlying(address _token, uint256 _amount) external {
-        _swapIntoUnderlying(_token, _amount);
+    function test_swapIntoUnderlying(address _token, uint256 _amount, uint256 slippage) external {
+        _swapIntoUnderlying(_token, _amount, slippage);
     }
 
-    function test_swapFromUnderlying(address _token, uint256 _amount) external {
-        _swapFromUnderlying(_token, _amount);
+    function test_swapFromUnderlying(address _token, uint256 _amount, uint256 slippage) external {
+        _swapFromUnderlying(_token, _amount, slippage);
     }
```

Comment:
maybe I'm missing something but I'm not certain what's the purpose of these tests, what are they asserting?

Reply:
not entirely sure tbh but IMO that problem is OOS & better addressed in another PR